docs/plugins.md

```diff
@@ -38,6 +38,12 @@ If the plugin is unable to be found automatically, you can load them with:
 * [`@prettier/plugin-php`](https://github.com/prettier/prettier-php)
 * [`@prettier/plugin-swift`](https://github.com/prettier/prettier-swift)
 
+## Unofficial Plugins
+
+* [@thorbenvh8/prettier-java](https://github.com/thorbenvh8/prettier-java)
```

Comment:
Please use the npm package names here.

Reply:
should I just remove the `@` or what?

the following work..

```
yarn add thorbenvh8/prettier-java
# or
npm install --save-dev thorbenvh8/prettier-java
```

Reply:
`prettier-plugin-java`: https://www.npmjs.com/package/prettier-plugin-java

Reply:
the other two are not on NPM and for the same reason even Python is not correct.. (https://www.npmjs.com/package/@prettier/python)

We can just point to the GitHub repositories until they are actually released, as we are looking for early adopters/testers and not the actual users.

Reply:
Yes, like to GitHub, but use the npm package names as the names.